src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/VepInputGeneratorStepTest.java

```diff
@@ -93,5 +109,6 @@ public void shouldGenerateVepInput() throws Exception {
 
         assertTrue(vepInput.exists());
         assertEquals("20\t60343\t60343\tG/A\t+", readFirstLine(vepInput));
+        assertEquals(EXPECTED_NON_ANNOTATED_VARIANTS, getLines(new FileInputStream(vepInput)));
     }
```

Comment:
Here the same method checkVepInputWasGenerated is called twice with different inputStudyId but then you are always checking the same result?